lib/msal-common/src/response/ResponseHandler.ts

```diff
@@ -133,16 +133,37 @@ export class ResponseHandler {
     /**
      * Function which validates server authorization token response.
      * @param serverResponse
+     * @param accessTokenRefresh
      */
     validateTokenResponse(
-        serverResponse: ServerAuthorizationTokenResponse
+        serverResponse: ServerAuthorizationTokenResponse,
+        accessTokenRefresh?: boolean
```

Comment:
nit: refreshAccessToken (verb>noun) seems more grammatical to me

Reply:
nit: another way to name bool is to use the verb `is` or the plural `are`. In this case `isTokenFromRefreshIn`

Reply:
I like `isTokenFromRefreshIn`, more relevant name